docs/sources/developers/kinds/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+draft: true
+title: Grafana schema
+weight: 200
+---
+
+# Grafana schema
+
+[Core kinds]({{< relref "core/_index.md" >}})
```

Comment:
Is the Core kinds link working for you? I had to remove the _index.md in my env:
`[Core kinds]({{< relref "core/" >}})`

Reply:
It works fine for me!